### ISSUE TITLE
Blinds state not correct

### ISSUE THREAD

**Issue by joconcepts** [pgerke/homebridge-freeathome-local-api#65]
**Describe The Bug:**
When closing a blind using the home app, the state changes to closing, however does never changes the state to closed. Clicking it again mostly fixes it. This also happens the other way around.
Another issue is when closing a blind via sensor click. The home app does not recognize any change. Once the blind is closed physically and you click the button in the home app again the state is set correctly.

**To Reproduce:**
Close a blind via home app and keep the app open. The blind button stays "closing".

**Expected behavior:**
State of blind in home app always correct.

Can anyone reproduce the issue? My guess is that the targetposition value has to be updated once the positionstate changes.

**Comment by DrMWeigand** [comment]
I can confirm that I also see this issue sometimes. Blinds are then displayed constantly as opening or closing. 

Another problem with blinds is that the state can be inverted. We have an awning for which a reversed state is displayed. The easiest option here would be to have an optional parameter to invert the state for the device.

<img width="950" alt="image" src="https://github.com/pgerke/homebridge-freeathome-local-api/assets/89643686/fac92122-0d66-4f4a-b96a-9a0cb170804b">

**Comment by Fridschi** [comment]
It's indeed still a problem. Here is a complete set of steps and screen shots that might help to reproduce. 

- Blinds are open. In this case my "Büro Rolladen"
![Step1](https://github.com/pgerke/homebridge-freeathome-local-api/assets/14573816/f1aea541-9a82-4839-856d-4c8667801e58)

- Blinds are then being closed through the home app. Works perfect and also recognises when the blinds are closed. Below screenshots from the home app and the logs from home bridge.
![image](https://github.com/pgerke/homebridge-freeathome-local-api/assets/14573816/c065eb81-43a8-406b-ae8d-9d970c10626a)
Corresponding log from home bridge looks ok too, although I do not understand the **PositionState -> 2**
![image](https://github.com/pgerke/homebridge-freeathome-local-api/assets/14573816/cad83564-e043-445c-a0ed-b5d34e0f6253)

- Next step is the confusing and obviously buggy part. I am using the physical switch in my room to open the blinds. While opening everything looks ok. The strange part happens, when the blind is closed. The status in the home app switches to closing "Schließen" and remains in this state. 
![image](https://github.com/pgerke/homebridge-freeathome-local-api/assets/14573816/4143e931-9a21-485c-b385-d998c55878ea)
The corresponding entries in the logs look like this:
![image](https://github.com/pgerke/homebridge-freeathome-local-api/assets/14573816/e6dff461-a9fd-41be-af2a-af094e0afede)

The timely sequence is:

1. I press the physical switch
2. PositionState is immediately set to 1 
3. The blinds are opening. The icon in the home app still shows closed "Schliessen"
4. Only once the blinds have entirely opened the Positionstate is set to 2
5. A second later Positionstate is set to 100
6. After Positionstate is set to 100, the icon in the home app switches to closing "Schliessen..."
7. If I tap on the blinds icon in the home app the state switches to opened "Geöffnet", which woul dbe the correct end state, if not being initiated manually by me.

Cheers
David
